crossing-training.lic

```diff
@@ -215,6 +218,17 @@ class CrossingTraining
     @cyclic_cycle_timer = Time.now
   end
 
+  def check_attunement
+    return unless @settings.crossing_training.include?('Attunement')
+    return unless DRSkill.getxp('Attunment') <= 33
+    return unless Time.now - @attunement_timer > @settings.attunement_interval
+    fput('perc')
```

Comment:
I can. However, from my observations it looked like bput was used when you needed to check the output of the command whereas fput is used when checking the result was not necessary. Is that not the case?